scripts/tag-web-features.ts

```diff
@@ -0,0 +1,112 @@
+/* This file is a part of @mdn/browser-compat-data
+ * See LICENSE file for more information. */
+
+import fs from 'node:fs/promises';
+import path from 'node:path';
+import { fileURLToPath } from 'node:url';
+
+import esMain from 'es-main';
+import { fdir } from 'fdir';
+import { features } from 'web-features';
+
+const dirname = fileURLToPath(new URL('.', import.meta.url));
+
+// Map from api.CoolThing.something to cool-thing
+const bcdToFeature = new Map();
+
+// Set of the first part of the BCD path encountered.
+const bcdDirs = new Set();
+
+for (const [feature, { compat_features }] of Object.entries(features)) {
+  if (!compat_features) {
+    continue;
+  }
+  for (const key of compat_features) {
+    bcdToFeature.set(key, feature);
+    bcdDirs.add(key.split('.')[0]);
+  }
+}
+
+const main = async () => {
```

Comment:
Maybe something like this?


```suggestion

// FixMe: The idea of scripts/bulk-editor is to have utilities for bulk updating BCD files programmatically. It also features an implementation to update tags which is not used here as the bulk-editor scripts aren't used and/or maintained properly at the moment.

const main = async () => {
```